pages/ppr-marketing.vue

```diff
@@ -5,45 +5,56 @@
         <v-container>
           <v-row>
             <v-col cols="6">
-              <h1 class="text--left">
+              <h1 class="text--left bigger-h1">
                 Register or search for legal claims on personal property
               </h1>
             </v-col>
           </v-row>
           <v-row>
             <v-col cols="5">
               The Personal Property Registry records security interests and
-              liens against personal property in British Columbia, whether the
-              property belongs to a business or individual.
+              liens against personal property belonging to British Columbia businesses
+              and individuals.
             </v-col>
           </v-row>
-          <v-row>
+          <v-row v-if="!loggedIn">
             <v-col cols="6">
               <SbcSigninButton />
             </v-col>
           </v-row>
-          <v-row>
+          <v-row v-if="!loggedIn">
             <v-col>
               New to BC Registries?
-              <a class="font-weight-bold" href="/"
+              <a class="font-weight-bold" href="https://dev.bcregistry.ca/business/auth/choose-authentication-method"
```

Comment:
you'll need to grab this link from the config: $config._

Reply:
done, I think

Reply:
I think you may have a double slash here - can you double check locally?

Reply:
there was & it's fixed